### PR TITLE
HRT-117 - Updated registered overview page

### PR DESCRIPTION
**Pull request by tony-gledhill**
Added background color to information table.
Added missing links
Added SVG icon to button link

### CODE REVIEW

**Review by tcmorris — APPROVED** [review]

File: pages/apply/index.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default function ApplicationHomePage(): JSX.Element {
         housing offer is made.
       </Paragraph>
 
-      <Announcement variant="info">
+      <Announcement variant="success">
```
Comment (tcmorris) [inline pages/apply/index.tsx:41]: The design looks to be a faded yellow, you can override with some custom styles if not available in design system.

File: pages/apply/index.tsx [hunk 2]
```diff
@@ -76,8 +76,34 @@ export default function ApplicationHomePage(): JSX.Element {
...
+          >
+            <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+          </svg>
+        }
+      >
+        Start now
+      </ButtonLink>
+
+      <Paragraph>
+        <Link href="#">Send me a link to apply later </Link>
```
Comment (tcmorris) [inline pages/apply/index.tsx:105]: Not sure we want to include these links if they don't do anything / go anywhere.

File: pages/apply/index.tsx (hunk 1 above)
Comment (tony-gledhill) [reply pages/apply/index.tsx]: I've confirmed with Chris that the background color needs to be blue as there were two versions of the design

File: pages/apply/index.tsx (hunk 2 above)
Comment (tony-gledhill) [reply pages/apply/index.tsx]: Chris has said this was added as part of a concept to see if users wanted to have a link sent to them to resume the process, but this is no longer needed. I've had this removed from the design and the code

**Pull request merged**